src/components/organisms/dialog-preview-image/index.js

```diff
@@ -6,24 +6,14 @@ import 'react-image-crop/dist/ReactCrop.css'
 import ImageCrop from '_molecules/image-crop'
 import Dialog from '_molecules/dialog'
 import Image from '_atoms/image'
-import Button from '_atoms/button'
 
-const DialogPreviewImage = ({
-  invalidProperties,
-  selectedFile,
-  crop,
-  withCrop,
-  onCancelClick,
-  ...props
-}) => (
+const DialogPreviewImage = ({ invalidProperties, selectedFile, crop, withCrop, ...props }) => (
```

Comment:
Did you spread the `...props` here because those are the dialog props that you need to waterfall from this component to the dialog?